ajgar_core/scripts/new_ik_solver.py

```diff
@@ -10,8 +10,13 @@
 import moveit_commander
 
 # custom imports
+rospack = rospkg.RosPack()
+pkg_file_path = rospack.get_path("plugin_pneumatic_gripper")
+script_import_path = pkg_file_path + "/scripts"
+sys.path.insert(1, script_import_path)
```

Comment:
Add a comment explaining this line of code. It would be great if you can add the source from which you took inspiration for this

Reply:
added comments

Reply:
Thanks